recipes/conda-replicate/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "conda-replicate" %}
+{% set version = "0.1" %}
+
+{% set package_name = name.replace("-", "_") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cswartzvi/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5432e410cecd5826df55a8ccafa056ae6c13508b13db3ef032182cd4b262e6e4
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - {{ name|lower }} = conda_replicate.cli:app
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+  run:
+    - python >=3.8
+    - conda >=4.12
+    - conda-build >=3.19
+    - networkx
+    - rich
+    - click >=8.0
+    - rich-click
+    - fsspec
+    - aiohttp
+    - pydantic
+    - requests
+    - pyyaml
+
+test:
+  imports:
+    - {{ package_name }}
+
+  commands:
+    - {{ name }} --help
```

Comment:
The jinja here obfuscate the recipe and doesn't really save that much typing. I'd rather see the form of the import and the CLI invocations.

Reply:
I was a little over-zealous with the templating. I have specified the import and CLI invocation directly.

Reply:
Yeah. It is tempting but for folks like me, who are constantly reading/modifying recipes, I prefer to use jinja vars only for the version or PyPI names. We rarely read the rendered recipe. We usually see the "raw" one on GH.